### PR TITLE
Workaround for Logstash 7 compatible keys

### PR DESCRIPTION
**Pull request by widhalmt**
* Add section in Readme about workarounds
* Add a workaround that will create PKCS8 keys with valid encryption which can be read by Logstash 7

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -225,6 +225,7 @@
             -in {{ ca_client_ca_dir }}/{{ inventory_hostname }}.key
             -topk8
             -passin pass:{{ ca_keypassphrase | default(omit, true) }}
+            {% if ca_ls7_workaround | bool %}-v1 PBE-SHA1-RC4-128{% endif %}
```
Comment (afeefghannam89) [inline tasks/main.yml:228]: How about putting the Cipher in a default variable? So the user can change it as he like.
`{% if ca_ls7_workaround | bool %}-v1 {{ ca_ls7_cipher }}{% endif %}`

Comment (afeefghannam89) [reply tasks/main.yml:228]: The ca_ls7_cipher should be defined under defaults.yml

**Review by afeefghannam89 — COMMENTED** [review]
I put one suggestion you can implement it or leave it.

File: tasks/main.yml (hunk 1 above)
Comment (widhalmt) [reply tasks/main.yml:228]: Great idea! I'll implement it

**Review by afeefghannam89 — APPROVED** [review]
Thanks Thomas :) it looks ok

**Pull request merged**